docs/worker-node/install-cvmfs.md

```diff
@@ -152,7 +152,7 @@ it.
     `cvmfs_cache_t`. This can be done by executing the following command:
 
         :::console
-        user@host $ chcon -R -t cvmfs_cache_t %RED%$CVMFS_CACHE_BASE%ENDCOLOR%
+        user@host $ chcon -R -t cvmfs_cache_t <CVMFS_CACHE_BASE>
```

Comment:
I think we could just instead delete the red color on this, and let people infer that they need to replace the value that they set for $CVMFS_CACHE_BASE for the variable.

Reply:
`<CVMFS_CACHE_BASE>` more clearly denotes that the user has to replace a value (especially since we note the angle brackets above). I don't think it's clear that users should replace `$CVMFS_CACHE_BASE` since it looks just like an environment variable.

Reply:
I don't feel strongly about it.